src/components/scene/xr-mode-ui.js

```diff
@@ -120,8 +120,6 @@ module.exports.Component = registerComponent('xr-mode-ui', {
 
     this.orientationModalEl = createOrientationModal(this.onModalClick);
     sceneEl.appendChild(this.orientationModalEl);
-
-    this.updateEnterInterfaces();
```

Comment:
I think this needs to be called so [AR], [VR] or both buttons are displayed based on dev configuration